install/action.yaml

```
@@ -5,6 +5,11 @@ inputs:
     description: Version of Telepresence to Install
     required: false
     default: latest
+  actionVersion:
+    description: Version of Telepresence to Install
```

It's sad that the `github.action_ref` is not available as an env var so we don't have to do something like this. I'd just change the description of the input, like:

```suggestion
    description: Placeholder input that holds the Telepresence github actions version. Do not use.
```